cylc/flow/simulation.py

```diff
@@ -76,7 +102,7 @@ def configure_sim_modes(taskdefs, sim_mode):
         )
 
 
-def get_simulated_run_len(rtc: Dict[str, Any]) -> int:
+def set_simulated_run_len(rtc: Dict[str, Any]) -> int:
     """Get simulated run time.
 
     rtc = run time config
```

Comment:
Docstring could be clearer

Reply:
How does "Calculate task run time in simulation mode from the simulation config"?